docs/docs/v7-upgrade-guidance.mdx

```diff
@@ -10,6 +10,196 @@ When doing destructive things like deleting a site, CLI for Microsoft 365 asks y
 
 Update your scripts to use the new option `force` instead of `confirm`. Verify that your scripts work as intended.
 
+## Updated `teams meeting list` command output
+
+In the past versions, the command `teams meeting list` returned calendar objects as result. Because of this, the IDs displayed in the output were not the same as the IDs used in the `teams meeting get` command and other teams meeting commands. In version 7 of the CLI for Microsoft 365, we have updated the output of the `teams meeting list` command to return meeting objects instead of calendar objects. This means that the IDs displayed in the output are now the same as the IDs used in the other `teams meeting` commands.
```

Comment:
```suggestion
In the past versions, the command `teams meeting list` returned calendar objects as a result. Because of this, the IDs displayed in the output were not the same as the IDs used in the `teams meeting get` command and other teams meeting commands. In version 7 of the CLI for Microsoft 365, we have updated the output of the `teams meeting list` command to return meeting objects instead of calendar objects. This means that the IDs displayed in the output are now the same as the IDs used in the other `teams meeting` commands.
```